raiden/tests/integration/rpc/test_assumptions.py

```diff
@@ -91,7 +78,7 @@ def test_transact_throws(deploy_client, blockchain_backend):
     deploy_client.poll(transaction)
 
     receipt = deploy_client.call('eth_getTransactionReceipt', data_encoder(transaction))
-    assert gas == quantity_decoder(receipt['gasUsed'])
+    assert 'status' in receipt and receipt['status'] == '0x0'
```

Comment:
this test is skipped, as the docstring said:

> This is not true anymore, after Byzantium a new opcode called REVERT was		
> introduced that stops the transaction without consuming all gas.

either leave it as it was or remove it

Reply:
hmm why? I did not notice the skip. I can unskip it, since we should check that indeed the receipt is correctly checked now. Why remove it?

Reply:
This test was written for checking this scenario (just realized the bad wording here, it should emphasize it's a transaction not just a call):

> call to a function that throws

This is covered by the following three tests:

- test_transact_throws_opcode
- test_transact_opcode_oog

Reply:
I see. And in those 2 tests through `check_transaction_threw` and the new change I added we do get the receipt check.

Then I guess it should be removed.